packages/ripple-nuxt-tide/modules/webform/mapping-filters.js

```diff
@@ -439,7 +447,8 @@ module.exports = {
               label: null,
               inlineLabel: element['#title'] ? element['#title'] : null,
               required: element['#required'],
-              validator: element['#required'] ? ['required'] : [],
+              validator: element['#required'] ? ['rplRequired', 'required'] : [],
```

Comment:
@elNico can this be actioned now or are you still waiting on something here? I am happy with the approach, but unsure if this is still WIP?

Reply:
@dylankelly I won't be able to work on it anymore since there's no budget. If you look through the file you can see quite a few specific fields or field groups like `address` for example that still just get the `required` validator applied:
https://github.com/dpc-sdp/ripple/pull/1074/files/68539de22f7435aa87253fb5d728362f3a65b37e#R290

It would mean that they would behave as always, returning "This field is required!", rather than anything more specific. I didn't want to apply `rplRequired` without actually testing it and didn't have a suitable form for everything. I guess the impact of using as is currently would simply be that it's not always consistent.

The main work is really creating a good test form that uses all fields that would get the additional validation, code-wise it's almost a find-and-replace.